BrowserSelect/Program.cs

```diff
@@ -146,7 +147,43 @@ public static string ProgramFilesx86()
             return Environment.GetEnvironmentVariable("ProgramFiles");
         }
 
+        public static bool DoesAutoRuleMatch(Uri uri, string pattern)
```

Comment:
Your method is not backward compatible and would break rules that work with the current version. Either add an indicator that would separate host-only rules from protocol+host+path rules and use different methods, or make sure it works the same way for host-only rules as the latest released version.
Example:
![image](https://user-images.githubusercontent.com/4184939/86516942-249ef800-be3a-11ea-9293-f96d0ed88c28.png)